jabgui/src/test/java/org/jabref/gui/newentry/NewEntryViewIdentifierDetectionTest.java

```diff
@@ -0,0 +1,55 @@
+package org.jabref.gui.newentry;
+
+import java.util.Optional;
+
+import org.jabref.logic.importer.CompositeIdFetcher;
+import org.jabref.model.entry.identifier.ArXivIdentifier;
+import org.jabref.model.entry.identifier.DOI;
+import org.jabref.model.entry.identifier.ISBN;
+import org.jabref.model.entry.identifier.Identifier;
+import org.jabref.model.entry.identifier.RFC;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class NewEntryViewIdentifierDetectionTest {
+
+    @Test
+    void detectsValidDOI() {
+        Optional<Identifier> result = CompositeIdFetcher.getIdentifier("10.1109/MCOM.2010.5673082");
```

Comment:
Better include the tests in `org.jabref.logic.importer.fetcher.CompositeIdFetcherTest` to have it more closerer to the method.

(Maybe, the whole `getIdentifier`  should move to org.jabref.model.entry.identifier.Identifier as `from(String identifier))